runtime/backtrace.c

```diff
@@ -374,3 +374,38 @@ CAMLprim value caml_get_current_callstack(value max_frames_value)
   caml_stat_free(callstack);
   CAMLreturn(res);
 }
+
+/* Print a callstack backtrace */
+void caml_print_raw_backtrace(value bt)
```

Comment:
If memory serves, this code was duplicated for simplicity (at the time we couldn't figure out how to reuse existing code). Suggestions how to do this would be welcome...

Reply:
This new code is a strict subset of what `caml_print_exception_backtrace` does, so you should be able to remove the duplication by calling the new function from `caml_print_exception_backtrace`?